vault/core.go

```diff
@@ -3767,3 +3770,7 @@ func (c *Core) GetRaftAutopilotState(ctx context.Context) (*raft.AutopilotState,
 
 	return raftBackend.GetAutopilotServerState(ctx)
 }
+
+func (c *Core) Events() *eventbus.EventBus {
```

Comment:
Is this used yet? I'd prefer we only add exported API surface area as it's actually getting used, as the surface area for `Core` is already massive.